app/controllers/publishers/vacancies_controller.rb

```diff
@@ -13,11 +13,28 @@ def show
     @vacancy = VacancyPresenter.new(vacancy)
   end
 
+  # rubocop:disable Metrics/AbcSize
   def index
     @selected_type = (params[:type] || :live).to_sym
```

Comment:
we should probably add a validating method in the controller even though there are  constraint the routes.rb. It's straight away visible that we used it with `send` and `public_send`

Reply:
same as above - this is just code motion. Brakeman couldn't see this code before for some reason, it was still there...